### DISCUSSION TITLE
hidden hides for both mobile and desktop view

### DISCUSSION THREAD

**Discussion by rhaber-tech** [tailwindlabs/tailwindcss#13749] [Help]
Hello! I have been trying to use tailwind to make a navbar and i am getting really confused with this hidden classname.  I have been referring to javascript mastery on youtube and at one point he uses ` <ul className="list-none sm:flex hidden justify-end items-center flex-1">` and the <li> tags following the ul are visible on the screen in desktop view and invisible in mobile view But in my case its not visible in  both the views .

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.

Otherwise, consider providing a project that reproduces the unexpected behavior.
